src/azure-cli-core/azure/cli/core/telemetry_upload.py

```diff
@@ -37,7 +78,7 @@ def upload(data_to_save):
         data_to_save = json.loads(data_to_save.replace("'", '"'))
     except Exception as err:  # pylint: disable=broad-except
         if in_diagnostic_mode():
-            sys.stdout.write('{}/n'.format(str(err)))
+            sys.stdout.write('ERROR: {}/n'.format(str(err)))
```

Comment:
Why 'Error'? If it's supposed to be there, it's fine with me.

Reply:
This is just for the sake of clear output.